cron/cron.sh

```diff
@@ -17,6 +17,7 @@ SOURCE="${BASH_SOURCE[0]}"
 input=$(dirname "$SOURCE")/projects.txt
 output=$(date +"%m-%d-%Y").json
 touch "$output"
+curl https://raw.githubusercontent.com/ossf/scorecard/main/cron/projects.txt >projects.txt
```

Comment:
>$input might be better ?

Reply:
I am going to skip these changes for this PR because I would have to test it again which will kill the existing cron job which is running. I would like it to finish to check its output.